### PR TITLE
Add sources page

### PR DESCRIPTION
**Pull request by wongcmichelle**
Changes
- Add sources page (it is the new page 2)
- Update appropriate .js files to .jsx (gets rid of `import React from "react";` is unused warning)
- Styling changes to centre text that goes over one line
- Update corner SVGs for less overlap (on smaller screens)

Screenshots (on smaller screen)
image
image

### CODE REVIEW

File: src/components/ProductBarInfoPage.jsx [hunk 1]
```diff
@@ -30,6 +30,10 @@ const useStyles = makeStyles({
     height: "100%",
     overflow: "hidden",
   },
+  description: {
+    maxWidth: "700px",
```
Comment (wongcmichelle) [inline src/components/ProductBarInfoPage.jsx:34]: I confirmed this maxWidth value with Sophia :)

File: src/components/SourceInfoPage.jsx [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+
+const useStyles = makeStyles((theme) => ({
+  img: {
+    display: "flex",
+    flexDirection: "column",
+    margin: "40px",
+  },
+  description: {
+    marginTop: "38px",
+    maxWidth: "700px",
```
Comment (wongcmichelle) [inline src/components/SourceInfoPage.jsx:16]: I confirmed this maxWidth value with Sophia :)

File: src/App.jsx [hunk 3]
```diff
@@ -110,9 +111,10 @@ function App() {
       <Box className={classes.page}>
         <Box className={classes.contentContainer}>
           {activeStep === 0 && <ScoringInfoPage />}
-          {activeStep === 1 && <ProductBarInfoPage />}
-          {activeStep === 2 && <CatalogueInfoPage />}
-          {activeStep === 3 && <FinishPage />}
+          {activeStep === 1 && <SourceInfoPage />}
+          {activeStep === 2 && <ProductBarInfoPage />}
+          {activeStep === 3 && <CatalogueInfoPage />}
+          {activeStep === 4 && <FinishPage />}
```
Comment (zhangium) [inline src/App.jsx:117]: The `handleSkip` function (line 101 above) needs to be updated as well. It should set the `activeStep` to be 4.

File: src/components/ProductBarInfoPage.jsx (hunk 1 above)
Comment (zhangium) [reply src/components/ProductBarInfoPage.jsx:34]: Great thanks for confirming!

File: src/components/SourceInfoPage.jsx [hunk 4]
```diff
@@ -0,0 +1,42 @@
...
+      <Typography variant="h5">
+        The product scoring is based on industry research
+      </Typography>
+      <Typography className={classes.description}>
+        The product scoring is based on industry research on how sustainability
+        is scored in cosmetic and fashion research. Cosmelo aimed to develop a
+        comprehensive score that judges sustainability of the entire life cycle
+        of products based on company policies and initiatives. The
+        sustainability categories are based on the 2018 release of the BPC
+        (Beauty and Personal Care) Sustainability Rating, simplified to 4
```
Comment (zhangium) [inline src/components/SourceInfoPage.jsx]: Can I request the link to the 2018 BPC Sustainability Rating doc be embedded? 

Link: https://www.sustainabilityconsortium.org/wp-content/uploads/2018/04/TSC%C2%AE-BPC-Product-Sustainability-Rating-System.pdf

**Review by zhangium — CHANGES_REQUESTED** [review]
Looks good! Just one change to handle the skip needed. My other comment is just an optional suggestion.

File: src/components/SourceInfoPage.jsx (hunk 4 above)
Comment (wongcmichelle) [reply src/components/SourceInfoPage.jsx]: Can you clarify what that means? do you mean you would like the text "BPC
        (Beauty and Personal Care) Sustainability Rating" to be a hyperlink?

File: src/styles/Colours.ts [hunk 5]
```diff
@@ -13,4 +13,5 @@ export enum Colours {
   Black = "#000000",
   White = "#FFFFFF",
   Grey = "#F9F9F9",
+  Link = "#0645AD",
```
Comment (wongcmichelle) [inline src/styles/Colours.ts:16]: Sophia said to use this colour for the link :)

File: src/components/SourceInfoPage.jsx (hunk 4 above)
Comment (zhangium) [reply src/components/SourceInfoPage.jsx]: I was thinking we could have the link be somewhere on the page, either like how we had it in our symposium poster or as a link with a footnote or something. Just thinking about transparency, thought we should provide a link/source when we have one.

**Review by zhangium — APPROVED** [review]
Thanks for making the changes! 💯

**Pull request merged**